hasura.planx.uk/migrations/1668434843791_run_sql_migration/up.sql

```diff
@@ -0,0 +1,6 @@
+CREATE OR REPLACE FUNCTION get_published_flow_version(pf published_flows)
```

Comment:
Instead of adding a new sql function and correlated computed field, wouldn't it be simpler to sort by `created_at`?